README.md

```diff
@@ -12,13 +12,14 @@ Install Dependencies (Recommended)
 
 #### create and activate qforte environment:
 ```bash
-conda create -n qforte_env python=3.7
```

Comment:
I understand removing `openfermion` and `openfermionpsi4`. What motivates the other changes?

Reply:
The reason we were requesting Python 3.7 was the OpenFermion dependency. Since we do not support OpenFermion anymore there is no reason to request for an older version of python. The problem of Psi4 not installing the latest version was also caused by Python 3.7. Using the latest version of Python, there is, as of now, no need to specify a particular version of Psi4.

When not installing OpenFermion and OpenFermion-Psi4, I got a bunch of errors when running the tests claiming that scipy was not recognized. I guess that scipy was installed as a dependency of OpenFermion. Now that we do not install OpenFermion, we need to install scipy. All of these changes stem from a series of trial and error experiments.